packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -72,14 +114,44 @@ export function SquadPageHeader({
           >
             {squad.name}
           </h1>
-          <h2
-            className={classNames(
-              'mt-1 text-center text-text-tertiary typo-body tablet:mt-2',
-              !shouldUseListModeV1 && 'laptopL:text-left',
+          <div className="mt-1 flex flex-row items-center justify-center text-text-quaternary tablet:mt-2 laptopL:justify-start">
+            <h2
+              className={classNames(
+                'text-center text-text-tertiary typo-footnote laptopL:text-left',
```

Comment:
```suggestion
                'text-center text-text-tertiary typo-footnote',
```
this is added below with the list layout condition